apps/web/src/app/[lang]/app/[type]/crm/companies/merchants/new/page.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
+import React from "react";
+import AutoForm from "@repo/ayasofyazilim-ui/organisms/auto-form";
+import { $UniRefund_CRMService_Merchants_CreateMerchantDto } from "@ayasofyazilim/saas/CRMService";
+import { createZodObject } from "src/utils";
+
+export default function Page() {
+  const schema =
+    $UniRefund_CRMService_Merchants_CreateMerchantDto.properties
+      .entityInformationTypes.items.properties.organizations.items;
+  const positions = Object.keys(schema.properties);
+  const formSchema = createZodObject(schema, positions);
+
+  return (
+    <div className="min-h-screen w-full bg-white">
```

Comment:
```suggestion
    <div className="w-full bg-white">
```

You need to use the layout logic here. Will merge this as first prototype. 
You need at the same time to use the submit button to show in the AutoForm. 

Here is the reference: 
https://github.com/vantezzen/auto-form?tab=readme-ov-file#submitting-the-form